README.md

```diff
@@ -267,6 +267,7 @@ DevOps is the combination of cultural philosophies, practices, and tools that in
 - [cAdvisor](https://github.com/google/cadvisor) - Analyzes resource usage and performance characteristics of running containers.
 - [ElastiFlow](https://github.com/robcowart/elastiflow) - Network flow monitoring (Netflow, sFlow and IPFIX) with the Elastic Stack.
 - [Co-Pilot](https://pcp.io/) - System performance analysis toolkit.
+- [Globalping CLI](https://github.com/jsdelivr/globalping-cli) - Run network commands like ping, traceroute and mtr from hundreds of global locations.
```

Comment:
Is it an early stage?

Reply:
Hey, the CLI tool is fresh but it's stable and prod ready. Plus it's auto-updatable on all OSes via package managers. The organization that is developing it is jsDelivr that has been around for 10+ years